app/views/candidate_interface/unsubmitted_application_form/show.html.erb

```diff
@@ -54,8 +54,8 @@
           <%= render(
             TaskListItemComponent.new(
               text: 'Add your references',
```

Comment:
Not essential, but I think this should be conditional too right? So 'Review your references' or 'Add your references' based on the application_references.present? boolean.